docs/source/contribute/index.rst

```diff
@@ -11,26 +11,22 @@ We welcome all kinds of contributions to EvaDB.
 Setting up the Development Environment
 =====================================================
 
-First, you will need to checkout the repository from GitHub and build EvaDB from
-the source. Follow the following instructions to build EvaDB locally. We recommend using a virtual environment and the pip package manager. 
+First, you will need to checkout the repository from GitHub and build EvaDB from the source. 
 
 .. code-block:: bash
 
    git clone https://github.com/georgia-tech-db/evadb.git && cd evadb
-   python3 -m venv test_evadb       # create a virtual environment
-   source test_evadb/bin/activate   # activate the virtual environment
-   pip install --upgrade pip         # upgrade pip
-   pip install -e ".[dev]"           # build and install the EvaDB package
-   bash script/test/test.sh          # run the EvaDB test suite
-   
-After installing the package locally, you can make changes and run the test cases to check their impact.
 
-.. code-block:: bash
+Follow the following instructions to build EvaDB locally. We recommend using a virtual environment and the pip package manager. 
 
-   pip install .         # reinstall EvaDB package to include local changes 
-   pkill -9 evadb_server   # kill running EvaDB server (if any)
-   evadb_server&           # launch EvaDB server with newly installed package
+.. code-block:: bash
 
+   python3 -m venv test_evadb_venv
+   source test_evadb_venv/bin/activate
+   pip install --upgrade pip
+   pip install -e ".[dev]"
```

Comment:
There are extra dependency besides [dev]. Shall we summarize them? Maybe we should have that information on the installation page and refer it here, since it will be helpful for users not only developers.

Reply:
Sure. That is a good idea.

Reply:
I think the installation page is no longer used. What would be a good place to show this information?

Reply:
I think we should add one.